python-threatexchange/threatexchange/cli/experimental_fetch.py

```diff
@@ -23,9 +25,15 @@ class ExperimentalFetchCommand(command_base.Command):
     """
 
     PROGRESS_PRINT_INTERVAL_SEC = 30
+    DEFAULT_REFETCH_SEC = 3600 * 24 * 85  # 85 days
 
     @classmethod
     def init_argparse(cls, ap) -> None:
+        ap.add_argument(
+            "--continuation",
+            action="store_true",
+            help="Fetch updates that you are missing, this overrides all other given options",
```

Comment:
So it wont necessarily use the same endtime - this is only true if there was an error and you are using the cursor/url that it failed at. If you are continuing from the last success it will query from that point until the end (i.e. so people can get periodic updates easily)

will go with "Continue fetching updates from where you last stopped with the same threat types. This will ignore all other options given"